guide/12-enrich-data-with-thematic-information/part6_country_enrich.ipynb

```diff
@@ -0,0 +1,1115 @@
+{
```

Comment:
<p>As in Part 1, we are looking at an example of a large retailer evaluating potential sites for a new location. This retailer is interested in using key criteria  to evaluate a few candidate sites. These criteria include competition, traffic, economic feasibility, and market potential for the areas surrounding the potential sites. Utilizing the GeoEnrichment module, the real estate site selection team can include demographic variables, such as lifestyle, income, spending, and education to understand potential customers in the study areas.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/1405/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>